holoviews/core/data/multipath.py

```diff
@@ -420,7 +421,7 @@ def values(cls, dataset, dimension, expanded=True, flat=True,
             if not len(dvals):
                 continue
             values.append(dvals)
-            if not is_points and expanded:
+            if (not is_points and expanded) or empty:
```

Comment:
This is not needed to get the example to work, but it makes a lot of sense to me.

Reply:
This will never actually execute because of the `len(dvals)` check above.

Reply:
Maybe that was why it made sense to me :upside_down_face: 

I have removed it.